src/vanilla/Model/MethodTS.cs

```diff
@@ -915,6 +748,34 @@ public void GenerateOperationSpec(TSObject operationSpec)
                 }
             }
 
+            operationSpec.ObjectProperty("responses", responses =>
+            {
+                bool isXml = CodeModelTS.ShouldGenerateXmlSerialization;
+                foreach (KeyValuePair<HttpStatusCode, Response> statusCodeResponsePair in Responses)
```

Comment:
`foreach((HttpStatusCode status, Response response) in Responses)`

Reply:
Fixed in https://github.com/Azure/autorest.typescript/pull/177/commits/f49ffa6c69fab6da9cdaa26f1b3f60b4bc4f794c.